sentry.edge.config.ts

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I'm not sure if we're planning on using Vercel Edge Runtime, if so this might be useful.

Reply:
[Preview Deployments](https://vercel.com/docs/concepts/deployments/preview-deployments) the Vercel thing I thought we might find useful, but I don't know enough about it to know if this is needed, happy to remove it for now if we're not sure.

Reply:
I don't think we'll use it since we already have a Mozilla Netlify account, and we'd need to hook it up to our own database anyway, so I'd lean towards removing it. But having it doesn't do much harm either, so I don't mind much either way :)

Reply:
Q: has Vercel been vetted and approved by IT/security or whomever does those things for external things?

Reply:
Probably not, so _if_ we were to use it (which I don't expect we will), we'll have to look into that.

Reply:
OK cool let's just remove it for now to avoid confusion then, thanks! It's easy to add back if we need it.